### ISSUE TITLE
Migration to WebApps WG

### ISSUE THREAD

**Issue by ayuishii** [WICG/cookie-store#186]
Is there any desire to move the Cookie Store API to WebApps WG?
The specification is a part of [WebApps WG charter](https://www.w3.org/2019/05/webapps-charter.html#wicgspecs), and will be shipping in [Chrome in M87](https://chromestatus.com/feature/5658847691669504).

Previous issue: https://github.com/WICG/cookie-store/issues/96 

@marcoscaceres @LJWatson

**Comment by marcoscaceres** [comment]
Sorry for the delay @ayuishii. In order to move it to WebAppsWG, we need support from a second implementer. I see Mozilla still has some concerns. Any word from the WebKit folks?

**Comment by ayuishii** [comment]
Thanks for the response @marcoscaceres ! I see, in that case we are still waiting for support. 
Currently [no strong signal](https://lists.webkit.org/pipermail/webkit-dev/2020-August/031365.html) from WebKit folks.

**Comment by marcoscaceres** [comment]
On a positive note, as it's already in the Web Apps WG charter, we can immediately take it once we get a second backer for it. It's not much, I know... but it's something 🙏

**Comment by marcoscaceres** [comment]
Any update on this with regards to implementer interest?

**Comment by markcellus** [comment]
Yes please... lets get this is. Because I'm growing tired of maintaining the polyfill for this 😆

**Comment by DCtheTall** [comment]
Since we have decided we want to move this to WHATWG, I am going to close this.
